components/backstage/components/backstage_sidebar.jsx

```diff
@@ -133,8 +138,10 @@ export default class BackstageSidebar extends React.Component {
             );
         }
 
+        // Note that we allow managing bot accounts even if bot account creation is disabled: only
+        // a permissions check is required.
         const botAccounts = (
-            <SystemPermissionGate permissions={['manage_bots']}>
+            <SystemPermissionGate permissions={['manage_bots', 'manage_others_bots']}>
```

Comment:
We don't ever grant `manage_others_bots` without `manage_bots`, but being thorough here (the logic is a boolean OR).